owlbot.py

```diff
@@ -54,10 +54,8 @@
 # This is required in order for s.copy() to work
 s._tracked_paths.add("googleapis")
 
-os.makedirs("google/api", exist_ok=True)
-
-s.copy("googleapis/google/api/annotations.proto", "google/api")
-s.copy("googleapis/google/api/http.proto", "google/api")
+# Copy the protos that we want to compile from googleapis
+s.copy("googleapis/google/cloud/orgpolicy/v1/*.proto", "google/cloud/orgpolicy/v1")
 
 # Clean up googleapis
 shutil.rmtree('googleapis')
```

Comment:
(Minor suggestion for pre-existing code; feel free to defer)

You have to keep this in sync with the lines above, right? So maybe capturing the protos in a list and running through the list in both places

```python
# Above:
additional_paths = ["googleapis"]
_ = [s._tracked_paths.add(dir) for dir in additional_paths]

# Here:
_ = [shutil.rmtree(dir) for dir in additional_paths]
```